frontend/app/routes/$lang/_protected/stub-sin-editor.tsx

```diff
@@ -136,6 +136,7 @@ export default function StubSinEditorPage() {
       {errorSummaryItems.length > 0 && <ErrorSummary id={errorSummaryId} errors={errorSummaryItems} />}
       <Form method="post" noValidate className="space-y-6">
         <InputField id="socialInsuranceNumberToStub" name="socialInsuranceNumberToStub" label={t('stub-sin-editor:index.edit-id-field')} required inputMode="numeric" pattern="\d{9}" placeholder="000000000" minLength={9} maxLength={9} />
+        <InputField id="userUUIDToStub" name="userUUIDToStub" inputMode="text" label={t('stub-sin-editor:index.UUID-label')} placeholder="00000000-0000-0000-0000-000000000000" />
```

Comment:
It would be nice to provide a dropdown of known SINs+UUIDs that is config driven. Not sure if that's something you'd want to try for this PR, though.